cost-analyzer/templates/cost-analyzer-network-costs-template.yaml

```
@@ -20,10 +27,13 @@ spec:
       {{- include "networkcosts.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.networkCosts.annotations }}
       annotations:
+        {{- with .Values.global.podAnnotations}}
         {{- toYaml . | nindent 8 }}
-      {{- end }}
+        {{- end }}
+        {{- with .Values.networkCosts.annotations }}
```

```suggestion
        {{- with .Values.networkCosts.podAnnotations }}
```

Should this be `podAnnotations`?

---

previously it was defined as `.Values.networkCosts.annotations` so did not change it. Right now `networkCosts` does not have `podAnnotations` property. Either we should remove this, or we can add a `podAnnotations` property to networkCosts

---

Ok, let's leave as is!